bucket/rsync.json

```diff
@@ -0,0 +1,30 @@
+{
+    "homepage": "https://itefix.net/cwrsync",
+    "version": "6.2.3",
+    "description": "rsync is a file transfer program capable of efficient remote update via a fast differencing algorithm.",
+    "license": "Itefix free software license (Simplified BSD)",
+    "##": "See also: https://www.itefix.net/content/cwrsync-licenseversion",
```

Comment:
```suggestion
    "license": "https://itefix.net/simplified-bsd-license",
```

Reply:
I believe we should also provide the URL of the license

Reply:
This is the license URL - https://itefix.net/simplified-bsd-license

Reply:
thanks. that part was copied from the old file at https://github.com/ScoopInstaller/Scoop/commit/dac9d8313dfafda25b58d4de8cf2409798bb7859

I've updated it